index.js

```diff
@@ -45,6 +46,34 @@ function filterResponseForImportantEvents(allEventsFromFetch) {
     return arrayOfImportantEvents;
 }
 
+function shouldIncludeEvent(eventType, association = 'OWNER') {
+    if (eventType.author_association !== association) {
```

Comment:
This is a bit of an odd function, in that it returns true if the if was false, and false if the if was true. 

In some cases, to help make it clearer what is going on, I'll extract a local. In this case, that might be (assuming we don't need the association parameter, per my other review comment): 
`const isAuthorAlsoTheOwner = (eventType.author_association === 'OWNER');`

Then the if would be `if (!isAuthorAlsoTheOwner) { return true; }`

Or, at that point, we could eliminate the if and just have:
`return !isAuthorAlsoTheOwner;`